srcpkgs/kodi/template

```diff
@@ -98,6 +92,16 @@ esac
 
 if [ -z "$CROSS_BUILD" ]; then
 	make_build_target+=" kodi-test"
+else
+	hostmakedepends+=" libmariadbclient-devel SDL2_image-devel lzo-devel"
+	hostmakedepends+=" libwaylandpp-devel"
+	makedepends+=" python3-devel ffmpeg-devel"
```

Comment:
Add `python3-devel` unconditionally to `makedepends`, `ffmpeg-devel` is there already.

Reply:
Done